### ISSUE TITLE
Request for Rootfs in Repository

### ISSUE THREAD

**Issue by FlyKOO** [Ananbox/ananbox#1]
I've tried building the rootfs, but I'm unable to run it successfully. Could you provide the correct rootfs.7z to ensure everything runs smoothly?

**Comment by TeddyNight** [comment]
OK, I will upload one working version later. But first, could you provide more information about your testing environment? And do you mean that you failed to build one rootfs?

**Comment by TeddyNight** [comment]
I"ve uploaded a arm64 version rootfs.7z, you can download and give it a try.

**Comment by FlyKOO** [comment]
Thanks for your help, but I'm still unable to enter the system. I tested it on my Xiaomi Mi 2S (Android 12) and got the proot.log and system.log files.
[proot.log](https://github.com/Ananbox/ananbox/files/14653677/proot.log)
[system.log](https://github.com/Ananbox/ananbox/files/14653681/system.log)

**Comment by TeddyNight** [comment]
Can you provide the logcat's log of the ananbox app and system error in logcat that contain keywords like `binder` or `ActivityManager` or `BroadcastReceiver`?

**Comment by FlyKOO** [comment]
Yes, I found some errors in the log, which contain these keywords.
[app_logs.log](https://github.com/Ananbox/ananbox/files/14668377/app_logs.log)
[all_logs.log](https://github.com/Ananbox/ananbox/files/14668385/all_logs.log)
![image (1)](https://github.com/Ananbox/ananbox/assets/86597661/405fa816-286b-4f94-b728-42b0b6becffc)

![unnamed](https://github.com/Ananbox/ananbox/assets/86597661/f0bc0f10-2f2f-49f3-a07a-272fb9cc4303)

![unnamed](https://github.com/Ananbox/ananbox/assets/86597661/b30df49c-df9b-41a9-8f3c-661fe37e605f)

**Comment by TeddyNight** [comment]
emmm, Ananbox doesn't support api level 32(Android 12L) currently because it require some extra work to figure out the format of the binder msg to get it work. You can try ananbox on api level 30 (Android 11) or api level 31 (Android 12, not Android12L)

**Comment by FlyKOO** [comment]
Okay🥳

**Comment by TeddyNight** [comment]
I've uploaded a new one to release page. You can download and test it.
